### PR TITLE
Add debian stretch

### PR DESCRIPTION
**Pull request by rollbrettler**

### CODE REVIEW

**Comment by rollbrettler** [comment]
@jlesage Do you think this change is valuable?

**Comment by jlesage** [comment]
Sorry it seems that this one fallen into the cracks.

While I agree to add  debian stretch, I'm not sure about the non-stable buster version.  Could you adjust the pull request to only include debian stretch?

**Comment by rollbrettler** [comment]
Is your concern the additional maintenance if buster breaks?

I see it as a preparation for the next release, but have no hard feelings about it, would just like to have more context.

Thanks.

**Comment by jlesage** [comment]
I prefer to offer baseimages based on stable releases.  Like you said, a development version may required more maintenance and are updated more often.  Also, from the point of view of someone who wants to build an application, I'm not sure why a baseimage based on a development distro would be taken over one based on a stable release.

**Comment by rollbrettler** [comment]
Fair enough, I updated the PR.

File: Dockerfile.debian [hunk 1]
```diff
@@ -26,7 +26,7 @@ COPY helpers/* /usr/local/bin/
 
 # Install s6 overlay.
 RUN \
-    add-pkg --virtual build-dependencies curl ca-certificates patch && \
+    add-pkg --virtual build-dependencies curl gnupg ca-certificates patch && \
```
Comment (jlesage) [inline Dockerfile.debian:29]: I think `gnupg` is not needed.  Could you remove it?

**Review by jlesage — CHANGES_REQUESTED** [review]
Before merging this, I just have a small change request.  Thanks!

**Comment by jlesage** [comment]
Closing this pull request, since we now have images based on debian 9.

**Pull request closed without merging**